docs/Versions.asciidoc

```diff
@@ -9,7 +9,7 @@
 release-state can be: released | prerelease | unreleased
 //////////
 
-:release-state:   prerelease
+:release-state:   unreleased
```

Comment:
This will break a lot of the existing docs, especially to do with installing 6.0.0-alpha2.  We can't do this until beta1 is out